addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -138,6 +151,11 @@ export class Chatbot extends Record {
         this.currentStep.completed = stepCompleted;
     }
 
+    async _processMultiLineAnswer(message) {
+        this.tmpAnswer = this.thread.composer.text;
+        return Promise.resolve().then(() => this._processAnswerDebounced(message));
+    }
+
```

Comment:
Nitpicking:
`_processX` methods are immediate processing of a chatbot step.
`_processMultiLineAnswer` debounces process answer again, so it's invoked again after some delay.
I wouldn't prefix with `_process` as to not mislead this is a branching that results to immediately something.

`_delayThenProcessAnswerAgain` (not the best naming, if you find better then feel free to change) gives a better understanding of its use without looking at implementation details.